src/main/java/net/dries007/tfc/objects/blocks/plants/BlockShortGrassTFC.java

```diff
@@ -54,14 +51,14 @@ public BlockShortGrassTFC(Plant plant)
         super(plant);
         if (MAP.put(plant, this) != null) throw new IllegalStateException("There can only be one.");
 
-        this.setDefaultState(this.blockState.getBaseState().withProperty(AGE, 0).withProperty(GROWTHSTAGE, CalenderTFC.Month.MARCH.id()));
+        this.setDefaultState(this.blockState.getBaseState().withProperty(DAYPERIOD, getDayPeriod()).withProperty(AGE, 0).withProperty(GROWTHSTAGE, plant.getStages()[CalenderTFC.Month.MARCH.id()]));
```

Comment:
Note: default states get assigned on construction. At this time it makes no sense to be calling `getDayPeriod()`, `CalendarTFC.getMonth()`, etc. as we are not even in a world yet. For clarity and simplicity sake these should all be constant values. (i.e. `CalendarTFC.getMonth()` -> `Months.MARCH.id()`) There are many places where this should be changed.